sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_utils/configurations.py

```diff
@@ -155,6 +157,10 @@ def _default_enable_live_metrics(configurations):
     configurations.setdefault(ENABLE_LIVE_METRICS_ARG, False)
 
 
+def _default_views(configurations):
+    configurations.setdefault(VIEWS_ARG, [])
```

Comment:
Seems Views defaults to ()  and is a Sequence. Could we change this to fit the OTel typing?

Reply:
nit